stripper/ze_stalker_ultimate_v4_9.cfg

```diff
@@ -65,4 +65,18 @@ modify:
 	{
 		"item_assaultsuit" "1"
 	}
-}
\ No newline at end of file
+}
+
+;Replace CD on Flash once items are upgraded after Bloodsucker is defeated
+modify:
+{
+	match:
+	{
+		"classname "logic_relay"
+		"targetname" "relay_bloodsucker_kill"
+	}
+	insert:
+	{
+		"OnTrigger" "point_servercommandCommandsm_setcooldown 550514 805-1"
```

Comment:
No item has the hammerid of 550514. Also I'd suggest you use the actual targetname as it's best practice, I only used point_servercommand in my Discord message as a non-map specific example.